light_dark_mode/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import Navbar from "./components/Navbar"
+import Main from "./components/Main"
+
+export default function App() {
+    const [darkMode, setDarkMode] = React.useState(false)
+
+    function toggleMode() {
+        setDarkMode(prevState => !prevState)
+    }
+
+    return (
+        <div className="container">
+            <Navbar darkMode={darkMode} toggleDarkMode={toggleMode}/>
```

Comment:
Please read https://airbnb.io/javascript/react/ - this is the coding style that we should use. In the Spacing section, they wrote "Always include a single space in your self-closing tag."
```
// bad
<Foo/>

// good
<Foo bar={baz} />
```
So please add a space before `/>` in all the files